### DISCUSSION TITLE
How to import media on GoAnimate Wrapper.

### DISCUSSION THREAD

**Discussion by TonytheTiger2003XD** [GoAnimate-Wrapper/GoAnimate-Wrapper#119] [General]
How do you import media such as props on GoAnimate Wrapper?

**Comment by ididthecorrupting** [comment]
he never responded...

**Comment by Windows81** [comment]
Hey.  Wrapper was inactive for a long time.  I might've forgotten to
respond.

If you're technically inclined, you can look at how modern revival projects
do this through their source code.
